lib/OpenQA/Utils.pm

```diff
@@ -642,6 +663,15 @@ sub render_escaped_refs {
     return bugref_to_href(url_to_href(xml_escape($text)));
 }
 
+sub find_bug_number {
+    my ($text) = @_;
+    my $marker = join('|', keys %bugrefs);
+    if ($text =~ /\S+\-(($marker)\d+)\-\S+/) {
```

Comment:
You're not using the second capturing group here.

Reply:
I am not sure if I understand your mean correctly. I just want to get $1 such as bsc123456,  so I think I don't need to use $2.

Reply:
Then don't use two capturing groups.

Reply:
I sense some terminological confusion: There are two sets of `()` here, so you get groups for `$marker` and `$marker\d+`.

Reply:
I will fix it. Thanks